README.md

```diff
@@ -46,7 +46,8 @@ To hit breakpoints, it is recommended to turn off the worker (in app settings).
 - `src/WebAssembly`: web-assembly host of the `App` (this is what's deployed online).
   - Trying out published version locally:
     ```ps1
-    dotnet tool install -g dotnet-serve
+    dotnet tool restore
+    dotnet workload restore
```

Comment:
Note: there is an overarching instruction above which says "For prerequisites, see `eng/build.sh` (in short: you need .NET SDK according to `global.json` and workloads `wasm-tools wasm-experimental`)." so I guess this `dotnet workload restore` instruction is a bit redundant, but at least it's self contained, so I'm fine keeping it.